pypeit/core/gui/identify.py

```diff
@@ -43,22 +43,34 @@ class Identify(object):
     file.
     """
 
-    def __init__(self, canvas, axes, spec, specres, detns, line_lists, par, lflag_color, slit=0):
+    def __init__(self, canvas, axes, spec, specres, detns, line_lists, par, lflag_color, slit=0, wv_calib=None):
         """Controls for the Identify task in PypeIt.
 
         The main goal of this routine is to interactively identify arc lines
         to be used for wavelength calibration.
 
-        Args:
-            canvas (Matploltib figure canvas): The canvas on which all axes are contained
-            axes (dict): Dictionary of four Matplotlib axes instances (Main spectrum panel, two for residuals, one for information)
-            spec (Line2D): Matplotlib Line2D instance which contains plotting information of the plotted arc spectrum
-            specres (dict): Three element list of Matplotlib Line2D/path instances, used for residuals plotting
-            detns (ndarray): Detections from the arc spectrum
-            line_lists (Table): Contains information about the line list to be used for wavelength calibration
-            par (WavelengthSolutionPar): Calibration parameters
-            lflag_color (list): List of colors used for plotting
-            slit (int): The slit to be used for wavelength calibration
+        Parameters
```

Comment:
Nice.  Thanks for getting the format right (I think).

Reply:
Haha... something I'm still working on  :-)